UQL/Interpreter.php

```diff
@@ -280,10 +282,15 @@ private function parseValue($value)
      */
     private function callFunction(ASTFunctionCall $functionCall)
     {
+        $functionName = $functionCall->getFunctionName();
+        $arguments = $functionCall->getArguments();
+
+        $arguments[] = new Context($this->dataSource->getEntityClass());
```

Comment:
Appending context at the end of list of argument allows us be BC.
But somehow it would look better for me to have context as a first argument (after all, it's always passed), but this would break BC.
What do you think?